wasm_webgpu_build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
let's use a single script wasm_build.sh  it could have a parameter:
```
./wasm_build.sh   //default: software
./wasm_builds.sh sw  //software engine
./wasm_build.sh wg //webgpu engine
```
something like:
```
if [ ! -d "./build_wasm" ]; then
    if webgpu
      sed "s|EMSDK:|$1|g" ./cross/wasm_webgpu.txt > /tmp/.wasm_cross.txt
      meson -Db_lto=true -Ddefault_library=static -Dstatic=true -Dloaders="all" -Dsavers="all" -Dthreads=false -Dbindings="wasm_beta" -Dengines="wg_beta" --cross-file /tmp/.wasm_cross.txt build_wasm
    else software
      sed "s|EMSDK:|$1|g" ./cross/wasm_x86_i686.txt > /tmp/.wasm_cross.txt
      meson -Db_lto=true -Ddefault_library=static -Dstatic=true -Dloaders="all" -Dsavers="all" -Dthreads=false -Dbindings="wasm_beta" -Dengines="sw" --cross-file /tmp/.wasm_cross.txt build_wasm
    fi
fi
```